resources/views/components/input/shortcode-textarea.blade.php

```diff
@@ -0,0 +1,53 @@
+@props([
+    'name' => 'body',
+    'enabled' => true,
+    'watermark' => '',
+    'maxLength' => 60000,
+    'initialValue' => '',
+])
+
+<?php
+
+$inputEnabled = $enabled ? '' : 'disabled';
+$buttonEnabled = $enabled ? ":disabled='!isValid'" : "disabled";
+
+$loadingIconSrc = asset('assets/images/icon/loading.gif');
+
+?>
+
+<div class='mt-4'>
```

Comment:
```suggestion
<div>
```

The component likely shouldn't have any self-applied layout styles like this on its container, otherwise reusability in different contexts becomes more difficult. Instead, this responsibility should belong to the parent:

```php
<div class="mt-4">
  <x-input.shortcode-textarea />
</div>
```

Reply:
done